skbio/stats/power.py

```diff
@@ -569,6 +618,12 @@ def bootstrap_power_curve(test, samples, sample_counts, ratio=None,
                           alpha=0.05, mode='ind', num_iter=500, num_runs=10):
     r"""Repeatedly calculates the power curve for a specified alpha level
 
+    .. note:: Deprecated in scikit-bio 0.2.3-dev
+    ``bootstrap_power_curve`` will be removed in scikit-bio 0.3.1. It is
+    depreciated in favor of using ``subsample_power`` or
```

Comment:
depreciated -> deprecated

The text underneath the `.. note::` line needs to be lined up with the `n` in `note` (required by numpydoc/Sphinx).